tests/Php80/PhpTokenTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Symfony\Polyfill\Tests\Php80;
+
+use PhpToken;
+use Symfony\Polyfill\Php80\PhpToken as PhpTokenPolyfill;
+use PHPUnit\Framework\TestCase;
+
+class PhpTokenTest extends TestCase
+{
+    public function testTokenize()
+    {
+        $code = \file_get_contents(__FILE__);
+        $tokens = PhpToken::tokenize($code);
+        $polyfillTokens = PhpTokenPolyfill::tokenize($code);
+        $this->assertEqualsCanonicalizing((array) $tokens[1], (array) $polyfillTokens[1]);
+        $this->assertEqualsCanonicalizing((array) $tokens[50], (array) $polyfillTokens[50]);
+        $this->assertEqualsCanonicalizing((array) $tokens[100], (array) $polyfillTokens[100]);
+        $this->assertEqualsCanonicalizing((array) $tokens[150], (array) $polyfillTokens[150]);
+        $this->assertEqualsCanonicalizing((array) $tokens[200], (array) $polyfillTokens[200]);
+        $this->assertEqualsCanonicalizing((array) $tokens[250], (array) $polyfillTokens[250]);
+    }
+
+    public function testGetTokenName()
+    {
+        // named token
+        $token = new PhpToken(\T_ECHO, 'echo');
+        $polyfillToken = new PhpTokenPolyfill(\T_ECHO, 'echo');
+        $this->assertEquals($token->getTokenName(), $polyfillToken->getTokenName());
+        // single char token
+        $token = new PhpToken(\ord(';'), ';');
+        $polyfillToken = new PhpTokenPolyfill(\ord(';'), ';');
+        $this->assertEquals($token->getTokenName(), $polyfillToken->getTokenName());
+        // unknown token
+        $token = new PhpToken(10000 , "\0");
+        $polyfillToken = new PhpTokenPolyfill(10000 , "\0");
+        $this->assertEquals($token->getTokenName(), $polyfillToken->getTokenName());
```

Comment:
let's use assertSame everywhere possible

Reply:
@nicolas-grekas what's reason? it's polyfill class and must be similar

Reply:
To ensure that types are also matching

Reply:
@nicolas-grekas ok